examples/operatorsource.cr.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: "marketplace.redhat.com/v1alpha1"
```

Comment:
Why is a new example required? Please point to the existing one in `deploy/examples`